std/file.d

```diff
@@ -401,11 +401,18 @@ version (Posix) private void[] readImpl(scope const(char)[] name, scope const(FS
 }
 
 
+// Hack to support unittesting "read" for file handles which do not support GetFileSize:
+version (Windows) private bool readImplTest = false;
```

Comment:
This won't do. Can the test create a pipe instead?

Reply:
I am going to try the pipe too !